README.md

```diff
@@ -585,12 +584,11 @@ Accelerated checkout buttons surface Apple Pay and Shop Pay options earlier in t
 
 ### Install the package
 
-Add the release candidate to your package manifest so you can import `ShopifyAcceleratedCheckouts` alongside `ShopifyCheckoutSheetKit`.
+Update your package manifest to import `ShopifyAcceleratedCheckouts` alongside `ShopifyCheckoutSheetKit`.
 
 ```swift
 dependencies: [
-  // Use the latest release candidate version
-  .package(url: "https://github.com/Shopify/checkout-sheet-kit-swift", "3.4.0-rc.3"..<"3.4.0")
+  .package(url: "https://github.com/Shopify/checkout-sheet-kit-swift", "3.4.0")
```

Comment:
This syntax is non-standard. The .package() method typically uses one of these forms:
  - .package(url: "...", from: "3.4.0")
  - .package(url: "...", exact: "3.4.0")
  - .package(url: "...", "3.4.0"..<"4.0.0")

Reply:
Good catch, I'll fix before we ship